kong-mesh/kong-mesh-sidecar-scc.yaml

```diff
@@ -7,9 +7,8 @@ allowPrivilegeEscalation: false
 allowPrivilegedContainer: false
 allowedCapabilities:
```

Comment:
again this is not a recommended strategy for openshift. the purpose of this demo was to provide a solution that aligns with Openshift

Reply:
I tried installing version 2.5 and version 2.6 of Kuma/Kong Mesh onto OpenShift using the current quickstart tutorial, and found some permission issues. I assume there may be permission changes after the tutorial was published. 

In addition, the permission for sidecars are different in the CNI mode and the "kuma-init" mode. We should also note this.

So we'll need to update this tutorial.

Reply:
Indeed the permissions are different. We can update the README to clarify you are intended to use cni, which is also enabled in the values file.

Reply:
Yeah, I think I misunderstood the original tutorial and I thought it would work with the "kuma-init" mode. If it was designed to be work only in the "kuma-cni" mode, then this whole PR may not be needed.

Reply:
ok so I think next steps are @jijiechen It would be great help updating the README here and a pr into the docs to clarify using "kuma-cni", and let me know if you're ok to close this PR. cc @jakubdyszkiewicz

Reply:
I'll close this and create a new one according to my latest update on the issue https://github.com/Kong/kong-mesh/issues/5607.